definitions/managed sevices.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Managed sevices
+status: Feedback Appreciated
+category: property
+---
+## Managed sevices
+### What it is
+ A complete safety net to your technical environment. It is the trust that you can have in a team to look after, both proactively and reactively, an ever-changing database environment. Above all, it is the knowledge that you have a team of experts to cover all areas of issues!
+
+Listed below are the main challenges Managed Services addresses and resolves.
```

Comment:
Great writing but not consistent with the current glossary style.

Reply:
read